src/context/AuthProvider.tsx

```diff
@@ -14,6 +14,7 @@ export const AuthProvider: React.FC<AuthProviderProps> = ({ children }) => {
   useEffect(() => {
     auth.onAuthStateChanged(user => {
       if (user) setCurrentUser(user);
```

Comment:
please add curly braces for if and else. 
Please also read https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces 
I think not using {} has some drawbacks